raml-200-tutorial.html

```diff
@@ -465,12 +467,14 @@ <h2 id="parameters">PARAMETERS</h2>
       body:
         application/json:
           type: &lt;&lt;resourcePathName|!singularize&gt;&gt;
+          example: |
+            &lt;&lt;exampleItem&gt;&gt;
       responses:
         200:
           body:
             application/json:
               example: |
-                { &quot;message&quot;: &quot;The &lt;&lt;resourcePathName|!singularize&gt;&gt; has been properly entered&quot; }
+                &lt;&lt;exampleItem&gt;&gt;
```

Comment:
I think it's ok to leave it as it was (with the message instead of returning the example). I think this was intended, to showcase the use of `{{resourcePathName|!singularize}}`